express/parsers/structure.py

```diff
@@ -41,6 +68,98 @@ def __init__(self, *args, **kwargs):
         self.lattice_only_structure = mg.Structure.from_str(self.structure_string, self.structure_format)  # deepcopy
         self.lattice_only_structure.remove_sites(range(1, len(self.structure.sites)))
 
+    def make_cart_file(self):
```

Comment:
This function creates the cartesian coord geom file that we need to generate the string that pybel can turn into an inchi.